### DISCUSSION TITLE
v1 docs

### DISCUSSION THREAD

**Discussion by DeveloperAlly** [Lilypad-Tech/lilypad-docs#4] [Lilypad General]
@levirybalov

I would love to know more about [the cryptographic approaches](https://docs.lilypadnetwork.org/research-and-vision/v1-documents/verifiable-computing) to verifiable computing... Any chance you'd have a good reference for a beginner to dig deeper on each of these methods?

**Comment by DeveloperAlly** [comment]
Unclear what an SLA is "Future approaches can incorporate SLAs and eventually support for homomorphic encryption to deal with this issue."

**Comment by levirybalov** [comment]
> @levirybalov
> 
> I would love to know more about [the cryptographic approaches](https://docs.lilypadnetwork.org/research-and-vision/v1-documents/verifiable-computing) to verifiable computing... Any chance you'd have a good reference for a beginner to dig deeper on each of these methods?

[Verifiable Computing Applications in Blockchain](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=9620108) is a good reference.

**Comment by levirybalov** [comment]
> Unclear what an SLA is "Future approaches can incorporate SLAs and eventually support for homomorphic encryption to deal with this issue."

SLA = Service Level Agreement. How exactly one would be agreed upon in this context isn't clear yet, but it seems like something resembling a traditional SLA should be achievable.
